### ISSUE TITLE
Don't use `use lib ...` is installable code

### ISSUE THREAD

**Issue by ugexe** [FCO/Red#467]
https://github.com/FCO/Red/blob/1a446a1de240f2754ac331397def05c2604bc6a8/bin/red#L2

The above is not an ideal thing to do. Once installed it will keep adding a `$PWD/lib` repository in the repo chain which means 1) it will have to recalculate repository ids for any repos later in the chain, which results in needing to precompile again when `$PWD` changes even though code is installed 2) potential issues where running `bin/red` in an e.g. maliciously crafted `$PWD` will load code.

In other words: 1) `use lib ...` is really for tests and not-installed scripts (not any modules or scripts that gets installed) -- give instructions to invoke with `-I ...` or install instead. 2) don't use relative paths to reference install-able things.

I'd also recommend not using `lib` and instead using `.` for the reasons explained in https://stackoverflow.com/questions/55661464/what-is-the-difference-between-i-and-ilib-in-perl6

**Comment by FCO** [comment]
thanks I'll fix that!
